content/blog/2020-05-05-updates-to-ragg-and-systemfonts/index.Rmarkdown

```diff
@@ -0,0 +1,169 @@
+---
+title: Updates to ragg and systemfonts
+description: |
+  We have recently updated some of our low-level graphics packages. Read on 
+  about what is new in ragg and systemfonts
+author: Thomas Lin Pedersen
+date: '2020-05-05'
+slug: updates-to-ragg-and-systemfonts
+categories:
+  - package
+tags:
+  - package
+  - ragg
+  - systemfonts
+authors:
+  - Thomas Lin Pedersen
+photo:
+  url: https://unsplash.com/photos/BVyNlchWqzs
+  author: Amador Loureiro
+---
+
+We're thrilled to announce the new releases of ragg and systemfonts. ragg is a package that provides a modern set of raster devices as alternatives to the ones you know from the grDevices package (e.g. `png()`). systemfonts is a low level package that provides font-related utilities to graphic devices, most importantly detecting and matching fonts to the correct font files across systems.
+
+While these are not the most flashy of releases, we thought it would be a good time to discuss some of our thoughts on the future of graphics in R, especially related to fonts.
+
+## ragg 0.2.0
+ragg is our current take on a modern set of raster devices for R. Together with the svglite package it provides alternatives to most of the devices that ships with base R. ragg is based on the AGG 2D graphics library, and provides both [high quality](https://ragg.r-lib.org/articles/ragg_quality.html) and [high performance](https://ragg.r-lib.org/articles/ragg_performance.html) output. One of its many virtues is that it provides high quality font support and automatically supports the different fonts installed on your system. This all happens through systemfonts as we'll discuss below.
```

Comment:
If you're going to sell these as alternatives to base R, I think you need also need to provide a compelling justification.

Reply:
I think we already sold it when we first released it. Should I just link to that or do you want me to write a full rehash?

Reply:
Either rehash, or weaken your language about base R.

Reply:
rereading this paragraph, I don't see any sort of strong language around base R? Can you point me towards it?

Reply:
Looking at it again, it doesn't feel too bad, but "provides alternatives to most of the devices that ships with base R" requires a bit more justification. I think you could just link to the original post.